eks_infrastructure/install_cluster_components.sh

```diff
@@ -33,4 +33,5 @@ CLUSTER_AUTOSCALAR_IMAGE_VERSION=${2}
 install_cluster_autoscalar ${CLUSTER_NAME} ${CLUSTER_AUTOSCALAR_IMAGE_VERSION}
 
 # install kubeflow
-../test/dlc_tests/eks/eks_manifest_templates/kubeflow/install_kubeflow.sh ${CLUSTER_NAME} ${AWS_REGION}
+OPERATION="INSTALL"
```

Comment:
Since this OPERATION is never checked for in the install script, is it needed? If someone were to run this like `install_kubeflow.sh <cluster>` , it would still install kubeflow without the need of the operation. Do we need to do any validity testing for this type of OPERATION?

Reply:
It is required because we are checking to make sure two arguments are passed.